pandas/core/computation/pytables.py

```diff
@@ -445,7 +447,7 @@ def visit_Attribute(self, node, **kwargs):
             # try to get the value to see if we are another expression
             try:
                 resolved = resolved.value
-            except (AttributeError):
+            except AttributeError:
                 pass
```

Comment:
Starting from Python 3.7 there is a context manager ``suppress``, which is useful for exactly such types of try/except/pass

https://docs.python.org/3.7/library/contextlib.html#contextlib.suppress

```
with suppress(AttributeError):
    resolved = resolved.value
```

@jreback, is pandas team using ``suppress``? Or will it introduce confusion only?

Reply:
not averse as it does provide nice readability -

Reply:
But we are still supporting `3.6` right? https://pandas.pydata.org/pandas-docs/stable/getting_started/install.html#python-version-support

Reply:
If these changes are to be assigned to 1.2 milestone, then there is a drop of Python 3.6 support (see https://github.com/pandas-dev/pandas/pull/35214).